test/LibraryManager.Mocks/LibraryInstallationState.cs

```diff
@@ -46,5 +46,7 @@ public class LibraryInstallationState : ILibraryInstallationState
         /// Indicates whether the library is using the default provider.
         /// </summary>
         public bool IsUsingDefaultProvider { get; set; }
+
+        public IReadOnlyList<FileMapping> FileMappings => throw new System.NotImplementedException();
```

Comment:
Documentation header?

Reply:
made them all inherit from the interface since there was nothing unique to this type.